lib/vtext.js

```diff
@@ -190,8 +398,50 @@ function processPixels(pixels, options, size) {
 }
 
 function vectorizeText(str, canvas, context, options) {
-  var size = options.size || 64
-  
+
+  var size = 64
+  var lineSpacing = 1.25
+  var styletags = {
+    breaklines: false, // Note: could also be turned on as new default
+    bolds: false,      // Note: could also be turned on as new default
+    italics: false,    // Note: could also be turned on as new default
+    subscripts: false, // Note: could also be turned on as new default
+    superscripts: true // Note: superscripts used to be partly supported
```

Comment:
We actually didn't support superscripts in this module. Anyway, what do you think @mikolalysenko @etpinard about the default behaviour of styling tags? Should we keep all of them off? Or if you prefer to enable any or the rest by default?